src/lib/app/streams.ts

```diff
@@ -456,3 +469,7 @@ export function isUserOwnerOfChannel(userDID: string, channel: ChannelInfo): boo
 export function isUserSubscribedToChannel(userDID: string, channel: ChannelInfo): boolean {
     return channel?.subscriberIds?.includes(userDID) && channel?.authorId !== userDID
 }
+
+export function hasUserRequestedSubscriptionToChannel(userDID: string, channel: ChannelInfo): boolean{
```

Comment:
So we are not including own channels into subscribed channels?

Reply:
No. The owner of a channel we filter by the authorId. On creation of a channel a Subscription is created but not included in the `subscribedIds` list this is only done when subscribing to other channels where the user is not the owner of.